net-core/Ical.Net/Ical.Net.UnitTests/DataTypes/RequestStatusTest.cs

```diff
@@ -126,56 +130,63 @@ private class Copyable : ICopyable
             public void CopyFrom(ICopyable obj) { }
         }
 
-        [Test]
-        public void CopyFrom_NonIRequestStatusTypeInput() 
+        private static IEnumerable<ITestCaseData> CopyFrom_InvalidTestCases()
         {
-            var requestStatus = new RequestStatus();
-            requestStatus.CopyFrom(new Copyable());
+            yield return new TestCaseData(new RequestStatus(), null).SetName("Request status CopyFrom() null");
+            yield return new TestCaseData(new RequestStatus(), new Copyable()).SetName("Request status CopyFrom() non-IRequestStatus type");
+        }
+
+        [Test, TestCaseSource(nameof(CopyFrom_InvalidTestCases))]
+        public void CopyFromTests(RequestStatus incoming, ICopyable copyable)
+        {
+            incoming.CopyFrom(copyable);
```

Comment:
I haven't evaluated the `CopyFrom` implementations, so I don't trust them. Leave it for now, though.